app/src/split/bluetooth/central.c

```diff
@@ -56,6 +57,7 @@ struct peripheral_slot {
 #if IS_ENABLED(CONFIG_ZMK_SPLIT_PERIPHERAL_HID_INDICATORS)
     uint16_t update_hid_indicators;
 #endif // IS_ENABLED(CONFIG_ZMK_SPLIT_PERIPHERAL_HID_INDICATORS)
+    uint16_t select_physical_layout_handle;
```

Comment:
IIUC, perhaps calling this `selected_*` would be more appropriate?

Reply:
Fair. Updated.